chainer/functions/loss/hinge.py

```diff
@@ -0,0 +1,75 @@
+import numpy
+
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+class Hinge(function.Function):
+
+    """Hinge loss."""
+
+    def __init__(self, use_cudnn=True):
```

Comment:
There are no CuDNN version of hinge function.  `use_cudnn` argument is not required.